### PR TITLE
[#11494] Make 'Download proof of submission' button more prominent

### LINKED ISSUE #11494: Make 'Download proof of submission' button more prominent
Current:
image

I suspect most users would click on the OK button without looking at the other button.

We can consider these tweaks:
1. Make the `Download ...` button green and move it to the right (as it is customary for the preferred option to be on the right)
2. Change color of the `OK` button to not green (e.g., use info/secondary colors)
3. Rename buttons as `Don't download proof-of-submission`, `Download proof-of-submission`

Other opinions welcome.

While we are at it, we can also bold (or highlight):
>You are **encouraged to download the proof-of-submission**, which will also contain your latest responses.

### PR DESCRIPTION
**Pull request by hoanglan21**
Fixes #11494

**PR Checklist**

Ensure that you have:
- [ ] Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr
  - [ ] Added the issue number to the "Fixes" keyword above
  - [ ] Titled the PR as specified in the abovementioned document
- [ ] Made your changes on a branch other than `master` and `release`
- [ ] Gone through all the changes in this PR and ensured that:
  - [ ] They addressed one (and only one) issue
  - [ ] No unintended changes were made
- [ ] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [ ] Added/updated tests, if changes in functionality were involved
- [ ] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

### CODE REVIEW

**Comment by hhdqirui** [comment]
@hoanglan21 Hi please make sure the checks are passed before we can review your PR.

File: src/e2e/java/teammates/e2e/pageobjects/AppPage.java [hunk 1]
```diff
@@ -212,7 +212,7 @@ public void verifyUnclickable(WebElement element) {
      */
     public void waitForConfirmationModalAndClickOk() {
         waitForModalShown();
-        WebElement okayButton = browser.driver.findElement(By.className("modal-btn-ok"));
+        WebElement okayButton = browser.driver.findElement(By.className("btn-info"));
```
Comment (halfwhole) [inline src/e2e/java/teammates/e2e/pageobjects/AppPage.java:215]: It looks like this line probably shouldn't be changed, it's responsible for the failure of some other E2e tests that depend on finding `modal-btn-ok` as well

File: src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html [hunk 2]
```diff
@@ -35,15 +35,15 @@ <h5 class="modal-title">
     Note that you can change your responses and submit them again any time before the session closes.
   </p>
   <p *ngIf="!hasFailToSaveQuestions">
-    You are encouraged to download the proof of submission, which will also contain your latest responses.
+    You are <b>encouraged to download the proof of submission</b>, which will also contain your latest responses.
   </p>
 </div>
 <div class="modal-footer">
   <button type="button" class="btn btn-info"
```
Comment (halfwhole) [inline src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html:42]: You could try adding an additional `modal-btn-ok` in the class here instead

**Review by halfwhole — COMMENTED** [review]
Also, could you attach a screenshot of the changes for reference? Thanks!

File: src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html [hunk 3]
```diff
@@ -35,15 +35,16 @@ <h5 class="modal-title">
     Note that you can change your responses and submit them again any time before the session closes.
   </p>
   <p *ngIf="!hasFailToSaveQuestions">
-    You are encouraged to download the proof of submission, which will also contain your latest responses.
+    <b>You are encouraged to download the proof of submission, which will also contain your latest responses.</b>
```
Comment (halfwhole) [inline src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html:38]: I noticed the position of the bold tags has changed, can we shift them to cover only `encouraged ... submission`?

File: src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html [hunk 4]
```diff
@@ -35,15 +35,16 @@ <h5 class="modal-title">
...
   </p>
 </div>
 <div class="modal-footer">
-  <button type="button" class="btn btn-info"
-          (click)="downloadProofOfSubmission()"
-          [disabled]="isAllQuestionSavingFailed">Download proof of submission</button>
   <button type="button" class="btn modal-btn-ok"
```
Comment (halfwhole) [inline src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html:42]: We could add an additional class (like perhaps `btn-info`) to the end here, that should remove the need for writing extra css

**Review by halfwhole — COMMENTED** [review]
Looks better! Just 2 comments:

**Comment by moziliar** [comment]
@hoanglan21 any update on this PR?

**Comment by madanalogy** [comment]
Hi @hoanglan21, if there's no update on this PR in 2 more days we'll be closing it for inactivity. Do let us know if you wish to proceed thanks

**Comment by jianhandev** [comment]
Closing due to inactivity, @hoanglan21 feel free to reopen if the issue is still open.

**Pull request closed without merging**